### PR TITLE
Reference RFC2119

### PR DESCRIPTION
**Pull request by bakerdb**
Added explanation re RFC2119 standards terminology.

### CODE REVIEW

File: docs/specification/1.0.md [hunk 1]
```diff
@@ -7,6 +7,9 @@
 
 The CDS Hooks specification describes the RESTful APIs and interactions between EHRs and CDS Services. All data exchanged through the RESTful APIs MUST BE sent and received as JSON structures, and MUST be transmitted over channels secured using Transport Layer Security (TLS). 
 
+### Terminology
```
Comment (brynrhodes) [inline docs/specification/1.0.md]: Suggest the term "Conformance Language" for this section, the word "Terminology" has a very different meaning in this context and is likely to be confused.

File: docs/specification/1.0.md [hunk 2]
```diff
@@ -7,6 +7,9 @@
...
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this specification are to be interpreted as described in [RFC2119](https://tools.ietf.org/html/rfc2119).
+
```
Comment (brynrhodes) [inline docs/specification/1.0.md]: Suggest using conformance language consistent with other HL7 specifications, including FHIR, as described in the [Conformance Language](http://build.fhir.org/conformance-rules.html#conflang) topic in the FHIR specification.

Comment (bakerdb) [reply docs/specification/1.0.md]: @brynrhodes  Good point.  Made change.

Comment (kpshek) [reply docs/specification/1.0.md]: I like the ability to use the full set of key words as described by rfc2119 as it is consistent with other IETF standards and gives us more verbal flexibility.

I'll discuss this with you @brynrhodes at the WGM and if there are still concerns, we can discuss further and open another issue,

**Pull request merged**